app/main/controllers/employee.py

```diff
@@ -0,0 +1,43 @@
+from flask import request
+from flask_restx import Resource
+
+from ..utils.dto import EmployeeDto
+from ..services.employee_service import (get_employee_by_id,
+                                         get_all_employees)
+
+from ..models.employee import Employee as EmployeeModel
+
+api = EmployeeDto.api
+_employee = EmployeeDto.employee
+
+# TODO: add argument parsing functionality. Note that Flasks's reqparse is depricated,
+# so should use marshmallow or similar instead.
+
+@api.route("",
+           doc={
+               "description": "Retreive a list of employees.",
+           })
+class Employees(Resource):
+    def get(self):
```

Comment:
Lets add the following here:
```python
@api.doc('List of employees')
@api.marshal_list_with(_employee)
```
This will give a name to the get endpoint for swagger and also tell it what the results will look like.